docs/content/guides/author-apps/dapr/how-to-dapr-building-block/index.md

```diff
@@ -0,0 +1,103 @@
+---
+type: docs
+title: "How-To: Add a Dapr building block"
+linkTitle: "Add a Dapr building block"
+description: "Learn how to add a Dapr building block to a Radius Application"
+weight: 300
+categories: "How-To"
+tags: ["Dapr"]
+---
+
+This how-to guide will provide an overview of how to:
+
+- Leverage a [Dapr building block](https://docs.dapr.io/developing-applications/building-blocks/) in your Radius Application
+
+## Prerequisites
+
+- [rad CLI]({{< ref "installation#step-1-install-the-rad-cli" >}})
+- [Radius Bicep VSCode extension]({{< ref "installation#step-2-install-the-vs-code-extension" >}})
+- [Radius environment]({{< ref "installation#step-3-initialize-radius" >}})
+- [Setup a supported Kubernetes cluster](https://docs.radapp.io/guides/operations/kubernetes/overview/#supported-clusters)
```

Comment:
These two are out of order. I'd remove the supported cluster piece since it's already a pre-requisite of the Radius Environment.